### ISSUE TITLE
High CPU usage when using leveldb

### ISSUE THREAD

**Issue by BlusteryS** [MemoriesOfTime/Nukkit-MOT#264]
# 🐞 Bug Feedback

## 📝 Bug Description
There is high CPU usage when using the leveldb format.

1. At the same time, the server does not start to lag; there are still **20 TPS** on the server
2. The load comes from constantly calling `doGarbageCollection` in LevelDB. With the anvil world format everything is fine
3. I tried turning use-native-leveldb on and off
4. The load does not disappear over time if you do not leave the server
5. When using anvildb the CPU load is 5-20% when flying, but in leveldb the load is 80-100% when staying in the same place
6. The load in status is displayed as 3%, but the processor is used to the maximum (**btop**/**htop** and others)

Here are the paths and methods where the 99% load occurs:
```java
Server#tickProcessor
-> Level#doGarbageCollection(long allocatedTime)
-> LevelDBProvider#doGarbageCollection(long time)
```

## 🔍 Steps to Reproduce
1. Use the leveldb world
2. Join the server

### 1. btop
![image](https://github.com/MemoriesOfTime/Nukkit-MOT/assets/52814854/89754c56-fc80-4ebc-830d-5beba15225de)
### 2. spark
![image](https://github.com/MemoriesOfTime/Nukkit-MOT/assets/52814854/27138cb0-ce00-4e77-b0b4-0e15d7b02c8c)
### 3. /status
![image](https://github.com/MemoriesOfTime/Nukkit-MOT/assets/52814854/2ccb68e3-2b12-41ee-a4d0-c32dd9696328)

## 💻 OS and versions
* OS: Linux
* Version: 1.20.80
* Java: OracleJDK and GraalVM
* Java version: 17 and 22

Tried it on two different servers in different locations and different characteristics

**Comment by lt-name** [comment]
StateBlockStorage compression has been temporarily canceled
At present, it seems that this function is not optimized very much, but it will take up a lot of performance.

**Comment by KoshakMineDEV** [comment]
Looks like fixes after #308
